### DISCUSSION TITLE
Unable to create the user because email is already taken and emails is invalid

### DISCUSSION THREAD

**Discussion by yuqingc31** [community/community#63806] [New to GitHub]
Hello,
I am creating a user by using my school's id and password for my group project, but when I entered my id and password, it shows "Unable to create the user because email [redacted due to PII] is already taken and emails is invalid". I have already removed the email from my GitHub setting, but I still get such error. 
Many thanks if you can provide helps.

**Comment by queenofcorgis** [comment]
Hi @yuqingc31,

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).  The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
